libs/gpg45-evaluator/src/main/java/uk/gov/di/ipv/core/library/gpg45/domain/CredentialEvidenceItem.java

```diff
@@ -192,4 +192,26 @@ public enum EvidenceType {
     public void setCredentialIss(String credentialIss) {
         this.credentialIss = credentialIss;
     }
+
+    public Integer getVerificationScore() {
+        if (isDcmaw()) {
+            if (checkDetails == null) {
+                return 0;
+            }
+
+            var checkMethodWithVerificationScore =
+                    checkDetails.stream()
+                            .filter(
+                                    checkMethod ->
+                                            checkMethod.getBiometricVerificationProcessLevel()
+                                                    != null)
+                            .findFirst();
+
+            return checkMethodWithVerificationScore.isPresent()
+                    ? checkMethodWithVerificationScore.get().getBiometricVerificationProcessLevel()
+                    : 0;
```

Comment:
This is fine, but if you want a couple of neater ways of writing this:

Streams:
```
return checkDetails.stream()
        .filter(checkDetail -> checkDetail.getBiometricVerificationProcessLevel() != null)
        .findFirst()
        .map(CheckDetail::getBiometricVerificationProcessLevel)
        .orElse(0);
```

Or old-school:
```
for (var checkDetail : checkDetails) {
    if (checkDetail.getBiometricVerificationProcessLevel() != null) {
        return checkDetail.getBiometricVerificationProcessLevel();
    }
}
return 0;
```